scalableminds/webknossos
# Also show measured distances in vx (in addition to nm)

### Todo

- [x] also add to other measurement tools (via right click, for all trees etc etc)
- [x] clean up so that there's no need to measure the tree twice (for nm and for vx)

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- use the following different measurement "tools":
  - tree-specific menu in trees-tab to measure tree length
  - "measure length of all skeletons" in trees tab
  - shift-rightclick a node and select "path length of this tree"
  - shift-rightclick a node and select "path length to this node"
  - shift-rightclick a node and see the bottom line of the context menu which shows the distance
  - shift-rightclick some position and see the bottom line of the context menu which shows the distance
- in all of the above cases, the distance should be shown in nm and in vx

### Issues:
- follow-up to #4898 

------
